client/components/Activity.js

```diff
@@ -91,7 +92,8 @@ const mapStateToProps = state => {
     comments: state.comments,
     type: state.activity.type,
     email: state.activity.user.email,
-    name: state.activity.product.name
+    name: state.activity.product.name,
+    activity: state.activity
```

Comment:
why are you pulling in the full activity file here when all the rest of the info is being destructured? Seems like a lot of unnecessary duplicate data